### ISSUE TITLE
Prevent unconnected users from editing

### ISSUE THREAD

**Issue by ms-studio** [prose/prose#1029]
I observed some non-technical users handling the prose.io platforms (their task was to make corrections in markdown files).

IMHO, the interface shouldn't present text files as editable when the user isn't logged and authorized. It's unintuitive that the user can edit files, without being able to actually save the changes. It's an invitation for data loss.

**Comment by dereklieu** [comment]
I agree. In the short term, something to show that you are not logged in, and therefore can't make edits, would be great. A more invasive fix would be to store the edits locally and then prompt login when you try to save. Would love help on either one.

**Comment by mikemorris** [comment]
I think there's an equivalent in CodeMirror, but a first pass could include something like adding the `readonly` attribute to `textarea` and `input` fields when a user isn't logged in. Unit tests for this functionality would be ideal to make sure it doesn't break the logged in functionality.

**Comment by waldyrious** [comment]
This would fix issue #643. The discussion there seemed to reach the same conclusion proposed here, so when this is done, both issues could be closed :)
